tests/network/BUILD.bazel

```diff
@@ -12,6 +12,7 @@ go_library(
         "services.go",
         "vmi_multus.go",
         "vmi_networking.go",
+        "vmi_networking_lifecycle.go",
```

Comment:
We are under the `network` folder, I do not think we need `networking` in the name.